### ISSUE TITLE
`WPS226` counts PEP 258 attribute docstrings

### ISSUE THREAD

**Issue by vyhuholl** [wemake-services/wemake-python-styleguide#3805]
With docstrings excluded from `WPS226` (#3804), functions, classes and modules are fine, but the attribute form still counts:

```python
class Some:
    first = 1
    """Some field docs."""

    second = 2
    """Some field docs."""

    third = 3
    """Some field docs."""

    fourth = 4
    """Some field docs."""
```

```
attr_doc.py:3:5: WPS226 Found string literal over-use: 'Some field docs.' 4 > 3
```

Same reasoning as #3803: extracting that string into a constant stops it from being documentation. PEP 258 calls it an attribute docstring and Sphinx `autodoc` picks it up.

It isn't covered by the fix for #3803 because that one defines a docstring as `body[0]` of the nearest module, class or function, which is what `is_doc_string()` in `logic/tree/strings.py` has always meant. An attribute docstring is the statement after an assignment, so it never matches.

I'm genuinely unsure this is worth fixing. This project documents its own module attributes with `#:` comments (147 of those lines in `wemake_python_styleguide/`), so the style guide doesn't obviously owe support to the other convention. But it's the same false positive as #3803, just one line lower.

If you do want it, the change looks narrow: also skip a string `ast.Expr` that directly follows an assignment in the same body. I can pick it up.

**Comment by sobolevn** [comment]
Yes, this would be good to have as well.
